appveyor.yml

```diff
@@ -4,100 +4,40 @@ clone_depth: 1
-    - C:\msys64\usr\bin\bash -lc "cd /c/%BUILD_FOLDER%/resources && sed -i 's/^\(FILEVERSION .*\)$/\1,0,"{build}"/' properties.rc"
-    - C:\msys64\usr\bin\bash -lc "cd /c/build_%ARCH_BIT% && make -j3"
-
-    # Copy the dependencies
-
-    - mkdir c:\%DEST_FOLDER%
-    - copy c:\%BUILD_FOLDER%\Scopy.exe c:\%DEST_FOLDER%\
-    - copy c:\%BUILD_FOLDER%\qt.conf c:\%DEST_FOLDER%\
-    - c:\msys64\%MINGW_VERSION%\bin\windeployqt.exe --dir c:\%DEST_FOLDER% --release --no-system-d3d-compiler --no-compiler-runtime --no-quick-import --opengl --printsupport c:\%BUILD_FOLDER%\Scopy.exe
-    - C:\msys64\usr\bin\bash -lc "cp -r /c/projects/scopy/resources/decoders  /c/%DEST_FOLDER%/"
-
-    - C:\msys64\usr\bin\bash -lc "tar -C /c/%DEST_FOLDER% --strip-components=3 -xJf /c/scopy-%MINGW_VERSION%-build-deps.tar.xz msys64/%MINGW_VERSION%/bin"
-    - C:\msys64\usr\bin\bash -lc "cd /%MINGW_VERSION%/bin ; cp -r libmatio-*.dll libhdf5-*.dll libszip*.dll libpcre*.dll libdouble-conversion*.dll libwinpthread-*.dll libgcc_*.dll libstdc++-*.dll libboost_{system,filesystem,atomic,program_options,regex,thread}-*.dll libglib-*.dll libintl-*.dll libiconv-*.dll libglibmm-2.*.dll libgmodule-2.*.dll libgobject-2.*.dll libffi-*.dll libsigc-2.*.dll libfftw3f-*.dll libicu{in,uc,dt}[!d]*.dll zlib*.dll libharfbuzz-*.dll libfreetype-*.dll libbz2-*.dll libpng16-*.dll libgraphite2.dll libjpeg-*.dll libsqlite3-*.dll libwebp-*.dll libxml2-*.dll liblzma-*.dll libxslt-*.dll libzip*.dll libpython3.*.dll libgnutls*.dll libnettle*.dll libhogweed*.dll libgmp*.dll libidn*.dll libp11*.dll libtasn*.dll libunistring*.dll libusb-*.dll libzstd*.dll /%MINGW_VERSION%/lib/python3.* /c/%DEST_FOLDER%/"
-    - mkdir C:\scopy_%ARCH_BIT%\.debug
-      #    - C:\msys64\usr\bin\bash -lc "/%MINGW_VERSION%/bin/objcopy -v --only-keep-debug /c/%DEST_FOLDER%/Scopy.exe /c/%DEST_FOLDER%/.debug/Scopy.exe.debug"
-    - C:\msys64\usr\bin\bash -lc "dump_syms /c/%DEST_FOLDER%/Scopy.exe > /c/%DEST_FOLDER%/Scopy.exe.sym"
-      #    - ps: C:\msys64\%MINGW_VERSION%\bin\strip.exe --strip-debug --strip-unneeded "C:\%DEST_FOLDER%\Scopy.exe"
-      #    - ps: C:\msys64\%MINGW_VERSION%\bin\objcopy.exe -v --add-gnu-debuglink="C:\%DEST_FOLDER%\.debug\Scopy.exe.debug" "C:\%DEST_FOLDER%\Scopy.exe"
-    - mkdir C:\%DEBUG_FOLDER%
-    - mv C:\%DEST_FOLDER%\Scopy.exe.sym C:\%DEBUG_FOLDER%
-    - mv C:\%DEST_FOLDER%\.debug C:\%DEBUG_FOLDER%
-
-    - 7z a "c:\scopy-%ARCH_BIT%bit.zip" c:\%DEST_FOLDER%
-    - appveyor PushArtifact c:\scopy-%ARCH_BIT%bit.zip
-    - 7z a "C:\debug-%ARCH_BIT%bit.zip" c:\%DEBUG_FOLDER%
-    - appveyor PushArtifact C:\debug-%ARCH_BIT%bit.zip
-    
+    - cmd: set PATH=%PATH%;"C:\Program Files (x86)\Inno Setup 5"
+    - cmd: C:\msys64\usr\bin\bash build_appveyor_mingw.sh
+    - sh: /home/appveyor/projects/scopy/install_deps.sh
```

Comment:
I think we can move this files (install_deps.sh, ...) from the top directory into another one just to keep things clean.
The files could be moved in a folder `ci/appveyor/`, we already have `ci/travis`

Reply:
We will move these files in the next PR (which will contain Appveyor flatpak build and Appveyor macOS build)